### PR TITLE
ML - model source type page

### PR DESCRIPTION
**Pull request by llali**
- Replacing card component with radio group card since it's deprecated
- Added a label for when a model source is selected
Screen Shot 2020-10-26 at 11 25 48 AM
Screen Shot 2020-10-26 at 11 26 40 AM

### CODE REVIEW

**Review by halerankin — APPROVED** [review]

File: extensions/machine-learning/src/common/constants.ts [hunk 1]
```diff
@@ -175,7 +175,12 @@ export const azureModels = localize('models.azureModels', "Models");
 export const azureModelsTitle = localize('models.azureModelsTitle', "Azure models");
 export const localModelsTitle = localize('models.localModelsTitle', "Local models");
 export const modelSourcesTitle = localize('models.modelSourcesTitle', "Source location");
-export const modelSourcePageTitle = localize('models.modelSourcePageTitle', "Where is your model located?");
+export const modelSourcePageTitle = localize('models.modelSourcePageTitle', "Select model source type");
+export const localModelSourceDescriptionForImport = localize('models.localModelSourceDescriptionForImport', "‘File Upload’ is selected. This allows you to import a model file from your local machine into a model database in this SQL instance. Click ‘Next’ to continue.​");
```
Comment (udeeshagautam) [inline extensions/machine-learning/src/common/constants.ts:179]: Might be a far fetched ask. Can we not make "‘File Upload’ is selected - a common string with the label being the argument?

File: extensions/machine-learning/src/views/models/modelSourcesComponent.ts [hunk 2]
```diff
@@ -110,29 +91,99 @@ export class ModelSourcesComponent extends ModelViewBase implements IDataCompone
...
 			}
 		});
-		this._flexContainer = modelBuilder.flexContainer()
-			.withLayout({
-				flexFlow: 'row',
-				justifyContent: 'space-between'
-			}).withItems(components).component();
+		let radioCardGroup = modelBuilder.radioCardGroup()
+			.withProperties({
+				cards: components,
```
Comment (udeeshagautam) [inline extensions/machine-learning/src/views/models/modelSourcesComponent.ts:96]: please check with ctrl + and - that this looks okay?

File: extensions/machine-learning/src/views/models/modelSourcesComponent.ts [hunk 3]
```diff
@@ -110,29 +91,99 @@ export class ModelSourcesComponent extends ModelViewBase implements IDataCompone
...
+				return 'registered';
+		}
+	}
+
+	private convertSourceIdToEnum(sourceId: string): ModelSourceType {
+		switch (sourceId) {
+			case 'local':
+				return ModelSourceType.Local;
+			case 'aml':
+				return ModelSourceType.Azure;
```
Comment (udeeshagautam) [inline extensions/machine-learning/src/views/models/modelSourcesComponent.ts:148]: no way to do ModelSourceType.Azure.tostring or equivalent?

**Review by udeeshagautam — APPROVED** [review]

**Comment by halerankin** [comment]
@llali I have minor changes to check in here for styles in modelSourcesComponent.ts

File: extensions/machine-learning/src/common/constants.ts (hunk 1 above)
Comment (llali) [reply extensions/machine-learning/src/common/constants.ts:179]: + @MsSQLGirl for feedback.

File: extensions/machine-learning/src/views/models/modelSourcesComponent.ts (hunk 2 above)
Comment (llali) [reply extensions/machine-learning/src/views/models/modelSourcesComponent.ts:96]: verified it look ok

**Pull request merged**